discord_bot.py

```diff
@@ -1,9 +1,30 @@
+import datetime
 import logging
 
+import discord
 from discord.ext import commands
 
+from personalities.base import Personality
+from personalities.bojack_horseman import BoJack
+from personalities.hank_hill import Hank
+
 
 class HuskieBot(commands.Bot):
+    current_personality = Personality
+
+    available_personalities = {
+        'default': Personality,
+        'hank': Hank,
+        'bojack': BoJack
+    }
+
+    last_personality_change = datetime.datetime.utcnow()
 
-    async def on_ready(self):
+    async def on_ready(self) -> None:
+        try:
+            await self.user.edit(avatar=None)
```

Comment:
Any reason wer're not setting the avatar here like we do in the change_personality command? [cogs/personality.py#L38](https://github.com/michaelhillebrand/huskiebot/pull/23/files#diff-caba36e032f2a9f4f77022de0f23951bR38)

Reply:
I will need to write something to have the personality to persist

Reply:
Resolved by adding a settings class in commit 7577f23